libs/ng-strapi-auth/src/lib/core/services/snack-bar/snack-bar.service.spec.ts

```diff
@@ -0,0 +1,22 @@
+import { TestBed } from '@angular/core/testing';
```

Comment:
Remember we should be able to override this service. Also I think the best option is to put it in the xevlabs utils library, as the xevlabsSnackBarLibrary